ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
The error for the example beginning on line 94 is confusing to me:
```ocaml
let f: ('a : void) -> 'b = fun x -> x # baz
[%%expect{|
Line 1, characters 36-37:
1 | let f: ('a : void) -> 'b = fun x -> x # baz
                                        ^
Error: Methods must have layout value.
       The layout of this expression is void, because
         of the annotation on the type variable 'a.
       But the layout of this expression must overlap with value, because
         it's an object.
|}];;
```
The confusing bit is the first line of this error: Where is there a method here that doesn't have layout value?  The error should be about the layout of the object `x`.

Reply:
Updated the first line to say "The object type must have layout value."